src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_params.py

```diff
@@ -112,6 +112,18 @@ def get_vm_size_completion_list(prefix, action, parsed_args, **kwargs):#pylint:
 register_cli_argument('vm extension image', 'type', CliArgumentType(options_list=('--name', '-n')))
 register_cli_argument('vm extension image', 'latest', CliArgumentType(action='store_true'))
 
+register_cli_argument('vmss extension', 'extension_name', name_arg_type, id_part='name', help='Name of extension')
+register_cli_argument('vmss extension', 'vmss_name', help='Scale set name')
+register_cli_argument('vmss extension', 'auto_upgrade_minor_version', CliArgumentType(action='store_true'))
```

Comment:
>CliArgumentType(action='store_true') [](start = 70, length = 36)

You don't need to put CliArgumentType when you are registering a parameter inline. It's an unnecessary anti-pattern we're trying to squash. You only need to do it when you plan to reuse it, in which case it should be assigned to a variable. Same with lines 119-121.